src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -86,6 +99,7 @@ public class GenerateProtoTask extends DefaultTask {
    *
    * Default: false
    */
+  @Internal
```

Comment:
"Attached to a task property to indicate that the property is not to be taken into account for up-to-date checking."
It probably should be taken into account for up-to-date checking.